spark-doris-connector/src/main/java/org/apache/doris/spark/rest/RestService.java

```diff
@@ -592,17 +650,20 @@ public static List<BackendV2.BackendRowV2> getBackendRows(SparkSettings sparkSet
     }
 
     /**
-     * choice a Doris BE node to request.
+     * benode does not exist, execute the original FE
+     * @param sparkSettings  settings
      * @param logger slf4j logger
-     * @return the chosen one Doris BE node
-     * @throws IllegalArgumentException BE nodes is illegal
+     * @return Doris Be nodes
+     * @throws DorisException
      */
-    @VisibleForTesting
-    public static String randomBackendV2(SparkSettings sparkSettings, Logger logger) throws DorisException {
-        List<BackendV2.BackendRowV2> backends = getBackendRows(sparkSettings, logger);
-        Collections.shuffle(backends);
-        BackendV2.BackendRowV2 backend = backends.get(0);
-        return backend.getIp() + ":" + backend.getHttpPort();
+    private static List<BackendV2.BackendRowV2> getBackendRowV2s(SparkSettings sparkSettings, Logger logger) throws DorisException {
+        List<BackendV2.BackendRowV2> backends = null;
+        if(notBeNode(sparkSettings, logger)){
+            backends = getBackendRows(sparkSettings, logger);
+        }else{
+            backends = getBeBackendRows(sparkSettings, logger);
```

Comment:
Is it enough to call the `getBeBackendRows` method directly in the `getBackendRows` method? Other methods should not need to be modified?

Reply:
Yeah, I should be able to take a look